Neos.Flow/Classes/Mvc/Routing/Dto/UriConstraints.php

```diff
@@ -229,6 +289,7 @@ public function withPathSuffix(string $pathSuffix, bool $prepend = false): self
      * Returns the URI path constraint, which consists of the path and query string parts, or NULL if none was set
      *
      * @return string|null
+     * @deprecated With Flow 7.0, use toUri()->getPath() instead. @see toUri()
```

Comment:
Does this really return the same value as `toUri()->getPath()`? If so, can we just replace the method body with that?

Reply:
Good call, I'll give that a try

Reply:
OK, I checked this again, but it's not exactly the same.
`toUri()->getPath()` will return a string.
This obsolete method returns a nullable string and it also adds the query string (as mentioned in the doc comment).
I adjusted the `@deprecated` annotation accordingly.

In general I don't think that this method is actually used outside of the core and there is no big harm in keeping it for a while. It's just not very beautiful :)